pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -581,6 +581,14 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 			},
 			errMsg: "invalid configuration: enableSystemLogHandler is required for enableSystemLogQuery",
 		},
+		{
+			name: "invalid configuration, resolverConfig not exist",
+			configure: func(conf *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration {
+				conf.ResolverConfig = "./test"
+				return conf
+			},
+			errMsg: "invalid configuration: resolverConfig with error open ./test: no such file or directory",
```

Comment:
Theoretically this test case should fail as actual error should have colon (':') after 'with error'.

Reply:
sorry for that, let me fix it.